mri/parallel_mri/extract_sensitivity_maps.py

```diff
@@ -215,8 +219,8 @@ def get_Smaps(k_space, img_shape, samples, thresh,
                                   endpoint=False)
                       for i in np.arange(np.size(img_shape)-1)]
         grid = np.meshgrid(*grid_space)
-        kspace_plane_loc, z_sample_loc, sort_pos = \
-            get_stacks_fourier(samples)
+        kspace_plane_loc, _, sort_pos, idx_mask_z = \
```

Comment:
Then do we need to have `z_sample_loc` returned in the `get_stacks_fourier` function?

Reply:
z_sample_loc is requiered in other call of the function get_stacks_fourier

Reply:
They both represent the same thing after all right? One is z_locations and other just indexes to these locations, can we open an issue to clean this function up? It would be great to get in this PR also, but that's your call..

Reply:
I raised an issue and will try to work on that for this PR